### PR TITLE
Server

### PR DESCRIPTION
**Pull request by danifosy**

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+        #!/bin/sh
```
Comment (marwinburesch) [inline .husky/pre-commit]: 🤔 hrm, what's this all about? Shouldn't be a problem, I'm just curious how this change happened

File: .eslintrc.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
   "env": {
     "browser": true,
-    "es2021": true
+    "es2021": true,
+    "node": false
```
Comment (marwinburesch) [inline .eslintrc.json]: this should be set to true

**Review by marwinburesch — CHANGES_REQUESTED** [review]
💯 
just ne little thing :)

File: .husky/pre-commit (hunk 1 above)
Comment (danifosy) [reply .husky/pre-commit]: I have no idea, maybe I wanted to talk in zoom and accidentally put the spaces here

**Review by marwinburesch — APPROVED** [review]
💯

**Pull request merged**